{{cookiecutter.project_name}}/.github/workflows/run-tests-fs.yml

```diff
@@ -0,0 +1,183 @@
+name: Feature and Training Integration Tests for {{cookiecutter.project_name}}
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - 'databricks-config/**'
+
+env:
+  DATABRICKS_HOST: {{cookiecutter.databricks_staging_workspace_host}}
+  NODE_TYPE_ID: {{cookiecutter.cloud_specific_node_type_id}}
+  {% if cookiecutter.cloud == "aws" -%}
+  DATABRICKS_TOKEN: {% raw %}${{secrets.STAGING_WORKSPACE_TOKEN}}{% endraw %}
+  {%- endif %}
+
+concurrency: {{cookiecutter.project_name}}-feature-training-integration-test-staging
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r test-requirements.txt
+      - name: Run tests with pytest
+        run: pytest
+  
+  integration_test:
+    needs: unit_tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        {%- if cookiecutter.cloud == "azure" %}
+      - name: Generate AAD Token
+        {% raw %}run: ./.github/workflows/scripts/generate-aad-token.sh ${{ secrets.STAGING_AZURE_SP_TENANT_ID }} ${{ secrets.STAGING_AZURE_SP_APPLICATION_ID }} ${{ secrets.STAGING_AZURE_SP_CLIENT_SECRET }}{% endraw %}
+        {%- endif %}
+      - name: Fetch integration test
+        uses: actions/github-script@v6
+        id: integration-test-content
+        with:
+          # TODO update the tasks and notebook parameters below to match your integration test setup.
```

Comment:
I wonder if we should document what this doing a bit in code comments, namely that we're building up the JSON payload to trigger the job run - it's a little tricky to infer from the step name "Fetch integration test"

Reply:
Added more information (also renamed the name of the step).